iOSIDPTemplate/Authenticator.xcodeproj/xcshareddata/xcschemes/Authenticator.xcscheme

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "0810"
```

Comment:
Will update these pieces on the `iOS` side with the latest from the templates in `dev`.